src/state/store.js

```diff
@@ -110,12 +106,16 @@ export const resetFilters = () => {
   store.setDimension('process', getDefaultFieldValue('process'));
 };
 
-export const probe = derived([probeSet, store], ([$probeSet, $store]) => {
-  if (!$probeSet || !$store.probeName) return undefined;
-  let pr = $probeSet.find((p) => p.name === $store.probeName);
-  if (CONFIG.transformProbeForGLAM) pr = CONFIG.transformProbeForGLAM(pr);
-  return pr;
-});
+export const probe = derived(
+  store,
+  ($store, set) => {
```

Comment:
```
  111:17  error  Expected to return a value at the end of arrow function  consistent-return
```

Reply:
I guess this isn't new but maybe we can fix it while we're here.

Reply:
Anonymous functions have to return now? It's wrapped in a code block so an implicit return shouldn't be expected unless I am missing something.

Reply:
No worries. If you leave this one alone I'll take care of it later. I think this might be a false positive.